crates/ide-completion/src/completions/dot.rs

```diff
@@ -26,16 +26,19 @@ pub(crate) fn complete_dot(
         item.add_to(acc, ctx.db);
     }
 
+    let is_field_access = matches!(dot_access.kind, DotAccessKind::Field { .. });
+
+    complete_fields(
+        acc,
+        ctx,
+        receiver_ty,
+        |acc, field, ty| acc.add_field(ctx, dot_access, None, field, &ty),
+        |acc, field, ty| acc.add_tuple_field(ctx, None, field, &ty),
+        is_field_access,
+    );
+
     if let DotAccessKind::Method { .. } = dot_access.kind {
         cov_mark::hit!(test_no_struct_field_completion_for_method_call);
```

Comment:
This cov mark makes little sense to be tested like this now (since the `else` branch is gone, which was what this mark wanted to assert is not being called)